# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/tar/fix_no_ascii_char.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./tests/genfile.c.orig	2003-07-27 23:44:12.000000000 -0400
+++ ./tests/genfile.c	2003-07-27 23:44:27.000000000 -0400
@@ -164,7 +164,7 @@
 	     stdout);
       fputs (_("\
 \n\
-Written by François Pinard <pinard@iro.umontreal.ca>.\n"),
+Written by Francois Pinard <pinard@iro.umontreal.ca>.\n"),
 	     stdout);
       exit (EXIT_SUCCESS);
     }
